### PR TITLE
Add arm64/aarch64 tests

### PR DESCRIPTION
**Pull request by Felixoid**
Changelog category (leave one):
- Build/Testing/Packaging Improvement

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
- Add arm64 packages
- Stream python logs in realtime with `PYTHONUNBUFFERED=1`
- Fix building docker images in docker/packager/packager script

### CODE REVIEW

File: docker/packager/binary/Dockerfile [hunk 1]
```diff
@@ -24,30 +24,25 @@ RUN apt-get update \
...
+        clang-tidy-${LLVM_VERSION} \
         cmake \
         curl \
+        fakeroot \
         gdb \
         git \
         gperf \
-        clang-${LLVM_VERSION} \
-        clang-tidy-${LLVM_VERSION} \
+        libicu-dev \
```
Comment (alesapin) [inline docker/packager/binary/Dockerfile:44]: Why we need them? We have bundled versions in contrib.

Comment (Felixoid) [reply docker/packager/binary/Dockerfile:44]: It looks like some strange merge back and forth issues to me. Should I just remove both readline and libicu?

Comment (Felixoid) [reply docker/packager/binary/Dockerfile]: Actually, just have it done. Let's see how it goes

Comment (Felixoid) [reply docker/packager/binary/Dockerfile]: https://s3.amazonaws.com/clickhouse-test-reports/32415/9a6e8dc61505b6bf9b27c61fc56e4fa257653aa6/fast_test__actions_.html

Fast tests passed, so maybe you are right

File: docker/packager/binary/Dockerfile [hunk 2]
```diff
@@ -66,15 +60,15 @@ ENV CC=clang-${LLVM_VERSION}
 ENV CXX=clang++-${LLVM_VERSION}
 
 # libtapi is required to support .tbh format from recent MacOS SDKs
-RUN git clone https://github.com/tpoechtrager/apple-libtapi.git \
+RUN git clone --depth 1 https://github.com/tpoechtrager/apple-libtapi.git \
```
Comment (alexey-milovidov) [inline docker/packager/binary/Dockerfile:63]: :+1:

**Review by alexey-milovidov — APPROVED** [review]

**Pull request merged**